app/src/main/java/com/duckduckgo/app/browser/favicon/FaviconManager.kt

```diff
@@ -64,14 +64,10 @@ class DuckDuckGoFaviconManager constructor(
     }
 
     override suspend fun loadToViewFromPersisted(url: String, view: ImageView) {
-        // avoid displaying the previous favicon when the holder is recycled
```

Comment:
Let's take a look at this together.

When you have more than 3 bookmarks, you can filter them. In that case, bookmarks without favicon could end up using the favicon from a recycled view.

Reply:
Talked about this via zoom and decided to rollback this change.